cfme/tests/control/test_actions.py

```diff
@@ -163,19 +175,16 @@ def _finalize():
     # Make it appear in the provider
     provider.refresh_provider_relationships()
 
-    if current_version() < "5.4":
-        # Get the SOAP object
-        soap = wait_for(
-            lambda: get_vm_object(vm_name),
-            message="VM object {} appears in CFME".format(vm_name),
-            fail_condition=None,
-            num_sec=600,
-            delay=15,
-        )[0]
-    else:
-        soap = None
+    # Get the SOAP/REST API object
```

Comment:
Is SOAP still referenced here? perhaps it needs removing from here and in the comment on line 163